### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by danielbldr**
Create microservice for SteadyMind to handle API calls to YouTube API

1. Create following endpoints: '/api/v1/sad', 'api/v1/happy', and 'api/v1/neutral'
2. These endpoints will return all videos from the SteadyMind YouTube playlist per category
3. Add fast_jsonapi to format JSON responses 
4. Added a category to each video based on if it's a yoga, meditation, or motivational video

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -0,0 +1,3 @@
+--require spec_helper
```
Comment (TravisBorgsmiller) [inline .rspec:1]: looks like we are using rspec instead of rack::test ?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+source 'https://rubygems.org'
```
Comment (TravisBorgsmiller) [inline Gemfile:1]: strong gemfile

File: .rspec (hunk 1 above)
Comment (SMJ289) [reply .rspec:1]: Good catch looks like we didn't end up needing that!

File: Gemfile (hunk 2 above)
Comment (SMJ289) [reply Gemfile:1]: Damn right it is!

**Pull request merged**